client/src/config/keys.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+CURRENCY_API_KEY: '231eb90015c1884b34ef94c0c0c3b4bc',
+IMAGE_UPLOAD_PRESET: 'i50l2x2r',
+IMAGE_API_SECRET: 'S253wP6tc6hhr2G69DzzWdpi7Gw',
```

Comment:
ummm ? i can see the API keys

Reply:
there is a way in create react app to make into an env variable [check this](https://medium.com/@tacomanator/environments-with-create-react-app-7b645312c09d)